packages/app/routes.tsx

```diff
@@ -28,57 +28,60 @@ import Plan from "./pages/Api/plan";
 import orgEdit from "./pages/Edit/organizationEdit";
 import OrganizationRegistration from "./pages/Registration/organizationRegistration";
 import IndividualRegistration from "./pages/Registration/individualRegistration";
+import OrgCatalog from "./pages/Organization/organizationCatalog";
 
 const routes = (
   <Route path="/" component={App}>
-    <IndexRoute component={Landing} />
+    {/*<IndexRoute component={Landing} />*/}
+    <IndexRoute component={OrgCatalog} />
 
-    <Route path="login" component={Todo} />
+    <Route path="login" component={Todo}/>
     <Route path="register">
-      <Route path="organization" component={OrganizationRegistration} />
-      <Route path="individual" component={IndividualRegistration} />
+      <Route path="organization" component={OrganizationRegistration}/>
+      <Route path="individual" component={IndividualRegistration}/>
     </Route>
+    <Route path="organizationCatalog" component={OrgCatalog}/>
```

Comment:
Can we get rid of this route if the base page is the organization catalog? Makes no sense to have the page in 2 locations.

Reply:
Yes, I think it would make the most sense to have it just at the default route for the site.